.github/actions/setup-poetry/action.yaml

```diff
@@ -8,17 +8,13 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - run: |
-        pip install --user pipx
-        pipx ensurepath
-      shell: bash
     - if: ${{ inputs.poetry-version == 'latest' }}
       run: |
-        pipx install poetry
+        pip install poetry
```

Comment:
Why pip rather than pipx?

Reply:
I found that on Apple Silicon, even thought pipx was installed in the previous step, pipx command isn't found here
[The failed build](https://github.com/PatrickDinh/algokit-cli/actions/runs/7884052914/job/21512425531)
I see that pipx is only used here to install poetry so I skip pipx.